### ISSUE TITLE
On Page refresh, slight blinking effect, not smooth grid-item load

### ISSUE THREAD

**Issue by zbayoff** [Shailu4u/ng-masonry-grid#18]
When using ng-masonry-grid, I find there is a small jumpy type effect when refreshing the page. This occurs in my app as well as your demo: https://ng-masonry-grid.stackblitz.io/ after you add a couple items and then refresh the page, there will be a jumpy, blinking effect. I would like to see a smooth rendering of the grid items.

Is there a solution you recommend to solve this?

**Comment by richjava** [comment]
Yes, it looks like the items render when loaded and then the animation kicks in, hides them and then fades them in again. I'd also like to know if there is a solution for this.
